task/jq/0.1/tests/print-result.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
```

Comment:
can you please move this before pipeline definition? Pipeline is not able to find this Task as it got applied post pipelinerun started

Reply:
Done!